modules/monitoring-listing-user-defined-alerting-rules.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="listing-user-defined-alerting-rules_{context}"]
+= Listing user-defined alerting rules
+
+As a cluster administrator, you can list user-defined alerting rules for core {product-title} platform monitoring.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+* To list all user-defined alerting rules in the `<openshift-monitoring>` namespace, run the following:
```

Comment:
`<openshift-monitoring>`
change to
`openshift-monitoring`

Reply:
fixed